### ISSUE TITLE
mCSB_container mCS_hidden mCS_no_scrollbar

### ISSUE THREAD

**Issue by Multi-Thinker** [malihu/malihu-custom-scrollbar-plugin#267]
my code is something like 

<pre>

(function($)
{$(window).load(function(){
$(".history").mCustomScrollbar()
;});
})(jQuery);

div class="mCustomScrollbar" data-mcs-theme="dark" 

div class="history"  
</pre>
but it always returns your scroll bar in <code>display:none</code> i can't display even when i try to change scroll shows but doesn't perform scroll functionality.. 
<pre>
 div class="mCSB_container mCS_hidden mCS_no_scrollbar"
</pre>

this always returns this...

**Comment by malihu** [comment]
Does .history element have a height set? 
I also replied to your comment on plugin homepage.

**Comment by Multi-Thinker** [comment]
Yes I have Set There height Too.. Very Earlier i tried almost many methods but all appears as error display:none, whenever i try to change that with fire bug or inspect element it instantly change again to none. Dont know what is wrong with this. Its a old issue.. since i post i am unable to use your plugin.

**Comment by malihu** [comment]
Can you send me a link or test page? I can't really tell what happens without seeing the complete code...

**Comment by sunil-ideas2it** [comment]
I am facing the same problem have u got the solution for this

**Comment by peasantCAPTAIN** [comment]
Can anyone figure out the problem with this situation?
